src/Venflow/Venflow/Database.cs

```diff
@@ -313,7 +310,7 @@ public TableBase<TEntity> Custom<TEntity>() where TEntity : class, new()
                     return new TableBase<TEntity>(this, (Entity<TEntity>)entity);
                 }
 
-                var entityBuilder = new EntityBuilder<TEntity>(this, string.Empty);
+                var entityBuilder = new EntityBuilder<TEntity>(_databaseConfigurationOptionsBuilder, string.Empty);
```

Comment:
And right here you can just query the `DatabaseConfigurationOptionsBuilder` from the `DatabaseConfigurationCache.DatabaseConfigurations` dictionary.